documentation/concepts/index.markdown

```diff
@@ -258,84 +194,40 @@ The depth image occupancy map updater includes its own _self-filter_, i.e. it wi
-
-
+Often, users may choose to implement their own kinematics solvers, e.g. the PR2 has its own kinematics solvers. A popular approach to implementing such a solver is using the [IKFast package](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/ikfast_tutorial.html) to generate the C++ code needed to work with your particular robot.
 
 * * *
 
-
-
-
-
 ## **Collision Checking**
 
-
-Collision checking in MoveIt! is configured inside a Planning Scene using the _CollisionWorld_ object. Fortunately, MoveIt! is setup so that users never really have to worry about how collision checking is happening. Collision checking in MoveIt! is mainly carried out using the [FCL](http://gamma.cs.unc.edu/FCL/fcl_docs/webpage/generated/index.html) package.
-
+Collision checking in MoveIt! is configured inside a Planning Scene using the _CollisionWorld_ object. Fortunately, MoveIt! is setup so that users never really have to worry about how collision checking is happening. Collision checking in MoveIt! is mainly carried out using the [FCL](http://gamma.cs.unc.edu/FCL/fcl_docs/webpage/generated/index.html) package - MoveIt!'s primary CC library.
 
 ### **Collision Objects**
 
-
 MoveIt! supports collision checking for different types of objects including:
 
-
-
-
   * Meshes
 
-
   * Primitive Shapes - e.g. boxes, cylinders, cones, spheres and planes
 
-
   * Octomap - the _Octomap_ object can be directly used for collision checking
 
-
-
-
-### **FCL**
-
-
-FCL, the _Flexible Collision Library_ is the primary collision checking library used in MoveIt!
-
-
 ### **Allowed Collision Matrix (ACM)**
 
-
-One of the important concepts to understand in MoveIt! is the _Allowed Collision Matrix_ or _ACM_. Collision checking is a very expensive operation often accounting for close to 90% of the computational expense during motion planning. The ACM encodes a binary value corresponding to the need to check for collision between two pairs of bodies (which could be on the robot or in the world). If the value corresponding to two bodies is set to 1 in the ACM, this specifies that a collision check between the two bodies is not needed. This would happen if, e.g., the two bodies are always so far way that they would never collide with each other.
-
-
+Collision checking is a very expensive operation often accounting for close to 90% of the computational expense during motion planning. The  _Allowed Collision Matrix_ or _ACM_ encodes a binary value corresponding to the need to check for collision between two pairs of bodies (which could be on the robot or in the world). If the value corresponding to two bodies is set to 1 in the ACM, this specifies that a collision check between the two bodies is not needed. This would happen if, e.g., the two bodies are always so far way that they would never collide with each other.
```

Comment:
I did not write that section, all I did was remove "One of the important concepts to understand in MoveIt! is" because I thought that was silly - the ACM is simply a minor optimization to reduce the number of collision checks, everything would work fine without it (just much much slower). 

But I see your point, seems like "the need to check for collision between pairs of bodies" is better